README.md

```diff
@@ -12,8 +12,11 @@ This repository contains Helm charts for the Okteto Cloud default repository:
 - [Movies](https://github.com/okteto/charts/tree/master/movies)
 
 
-# How to add a new chart
+# How to submit a new chart
 
-- Add your chart source code
+- Fork https://github.com/okteto/charts.
+- Add your chart source code in a new folder (Review https://okteto.com/docs/cloud/repos.html to see how to make your chart Okteto Cloud friendly). 
+- Test your chart with Okteto Cloud (Feel free to [reach out](https://twitter.com/oktetohq) if you need help).
 - Run `helm package chart-name` where `chart-name` is the folder of your new chart
-- Run `helm repo index --url https://apps.okteto.com/ --merge index.yaml .`
```

Comment:
@pchico83 how about we create a github action that does this?

Reply:
Sounds goo